recipes/full_finetune_distributed.py

```diff
@@ -681,7 +735,12 @@ def train(self) -> None:
                         time_per_step = time.perf_counter() - t0
                         log_dict = {
                             "loss": loss_to_log,
-                            "lr": self._optimizer.param_groups[0]["lr"],
+                            "lr": get_lr(
```

Comment:
combine `get_lr` as an utils for both distributed and single_device to validate if all the LR are the same and return if True